<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QTTODOClass</class>
 <widget class="QMainWindow" name="QTTODOClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>746</width>
    <height>566</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FocusLock</string>
  </property>
  <property name="windowIcon">
   <iconset resource="QTTODO.qrc">
    <normaloff>:/QTTODO/tomato_Icon~1.ico</normaloff>:/QTTODO/tomato_Icon~1.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,3">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>功能页</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="nowtmtButton">
         <property name="text">
          <string>即时番茄</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="timetmtButton">
         <property name="text">
          <string>添加定时番茄</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="editButton">
         <property name="text">
          <string>编辑选中番茄</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="delButton">
         <property name="text">
          <string>删除选中番茄</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="whiteappButton">
         <property name="text">
          <string>白名单应用设置</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>番茄事项</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTableWidget" name="tmtTableWidget">
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="frameShape">
          <enum>QFrame::Shape::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Sunken</enum>
         </property>
         <property name="lineWidth">
          <number>1</number>
         </property>
         <property name="midLineWidth">
          <number>0</number>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
         </property>
         <attribute name="verticalHeaderStretchLastSection">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>746</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QTTODO.qrc"/>
 </resources>
 <connections/>
</ui>
